### PR TITLE
Remove azure-iot-edge plugin from distribution

### PR DESCRIPTION
**Pull request by formulahendry**

### CODE REVIEW

**Comment by formulahendry** [comment]
@daniel-beck Please help review. Thanks!

**Comment by daniel-beck** [comment]
@formulahendry Could you provide more details than this? Thanks!

**Comment by formulahendry** [comment]
Hi @daniel-beck , Azure IoT Edge Plugin is deprecated, so we want to unpublish it.
And I have added an entry in the deprecations file.
Please help review. Thanks a lot!

File: resources/deprecations.properties [hunk 1]
```diff
@@ -12,6 +12,7 @@ appio = https://git.io/JT9ZT
 # https://wiki.jenkins.io/display/JENKINS/AppThwack+Plugin
 appthwack = https://wiki.jenkins.io/x/cwchB
 assembla-oauth = https://groups.google.com/d/msg/jenkinsci-dev/TVz-D5etsUM/Knx9zXtEnSwJ
+azure-iot-edge = https://git.io/JtOjd
```
Comment (daniel-beck) [inline resources/deprecations.properties:15]:
```suggestion
# https://github.com/jenkinsci/azure-iot-edge-plugin/blob/master/Readme.md
azure-iot-edge = https://git.io/JtOjd
```

**Review by daniel-beck — APPROVED** [review]
Thanks, that's helpful.

Just a tiny change on top to make this easier to read in the future, and we're good.

**Comment by formulahendry** [comment]
Hi @daniel-beck , thanks a lot for helping the review! Would you please help merge this PR? Thanks!

**Comment by daniel-beck** [comment]
@formulahendry Thanks for the reminder, I've done it now. I was waiting for the PR build to finish.

**Comment by formulahendry** [comment]
Thanks @daniel-beck !
BTW, I still could see the plugin: https://plugins.jenkins.io/azure-iot-edge/
Is there any time delay for this plugin to be unpublished from the Jenkins plugin marketplace?

**Comment by daniel-beck** [comment]
Caches be caching. It's gone from http://updates.jenkins.io/download/plugins/ which is always up to date with update site contents; it's no longer being offered to Jenkins admins for install.

**Comment by MarkEWaite** [comment]
See [Jenkins docs mailing list discussion](https://groups.google.com/g/jenkinsci-docs/c/MRpUHjdmkzA/m/Xvi0A38LBgAJ) investigating why the pipeline steps reference has not been removed.

**Pull request merged**